_transactions.md

```diff
@@ -1080,3 +1083,53 @@ Besides the limits listed there, it's also recommended to preemptively [check a
   We currently do not expose an endpoint that lists all the limits that would apply for a given transaction.
   Your application should be prepared to handle these limit errors and forward the relevant information to the user so they can adjust the transaction properties accordingly.
 </aside>
+
+## Transacting with credit/debit cards
+
+The card details provided in this portion of the documentation are not real cards but specifically designed for Sandbox transactions. These can be used to force specific outcomes and test the various flows pertaining to using card data to perform transactions.
+
+<aside class="notice">
+  Any value entered for <i>expiry date</i> and <i>CVV</i> will be accepted as valid when used in conjunction with the card data listed in this section.
+</aside>
+
+### Credit/debit card deposits and withdrawals
+
+The following list contains specific transaction amount values that are meant to trigger a target error when trying to perform deposits or withdrawals on any given card account.
+
+Amount | Error                 |
+------ | --------------------- |
+12.12  | card_unauthorized     |
```

Comment:
This will probably not work for an API user. Internally, for dev environments, we have a different plan which has no fees applied, therefore the amount the user inputs on the form is the same that reaches the gateway. For the average user in sandbox this is not the case as the `individual` plan has fees associated which will be applied to the value and the end result, and the value that will reach the gateway, will not be the same the user inputs on the form.

Reply:
So, in effect, what you are suggesting is for us to drop this table, correct?

Reply:
The decision in conversation between Ayush, Freitas and myself was to drop this table.